collections/_article/release-candidate-godot-4-3-rc-1.md

```diff
@@ -0,0 +1,121 @@
+categories: ["pre-release"]
+author: "Rémi Verschelde"
+image: /storage/blog/covers/release-candidate-godot-4-3-rc-1.webp
+image_caption_title: "ColdRidge"
+image_caption_description: "A game by Frog Collective"
+date: 2024-07-25 15:00:00
+---
+
+We are entering the final stage of development for Godot 4.3, which is the [Release Candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate): all features are in-place, the most critical regressions have been tackled, and so we're confident that it's now ready for general use in the vast majority of cases...
+
+... which is usually the cue for more testing to jump aboard the pre-release train, test this release candidate on their projects, and ~shatter our hopes~ confirm that everything is ready for prime time!
+
+More seriously, Godot is a big piece of software and it's hard for contributors and even unit tests to validate all areas of the engine when developing new features or bug fixes. So we rely on extensive testing from the community to find engine issues while testing dev, beta, and RC snapshots in your projects, and reporting them so that we can fix them prior to tagging the 4.3-stable release.
+
+Please, consider [supporting the project financially](https://fund.godotengine.org), if you are able. Godot is maintained by the efforts of volunteers and a small team of paid contributors. Your donations go towards sponsoring their work and ensuring they can dedicate their undivided attention to the needs of the project.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.3.rc1/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+---
+
+*The cover illustration is from* [**ColdRidge**](https://store.steampowered.com/app/3052500/ColdRidge/-), *a turn-based exploration game set in the mystical Wild West, developed by [Frog Collective](http://frog-collective.com/en/) with Godot 4. The game was revealed in early July and just got a playable demo [on Steam](https://store.steampowered.com/app/3052500/ColdRidge/), where you can also wishlist it.*
+
+## Highlights
+
+We covered the most important highlights from Godot 4.3 in the previous [4.3 beta 1 blog post](/article/dev-snapshot-godot-4-3-beta-1/), so if you haven't read that one, have a look to be introduced to the main new features added in the 4.3 release.
+
+Especially if you're testing 4.3 for the first time, you'll want to get a condensed overview of what new features you might want to make use of.
+
+This section covers changes made since the previous [beta 3 snapshot](/article/dev-snapshot-godot-4-3-beta-3/), which are mostly regression fixes, or "safe" fixes to long standard issues.
```

Comment:
```suggestion
This section covers changes made since the previous [beta 3 snapshot](/article/dev-snapshot-godot-4-3-beta-3/), which are mostly regression fixes, or "safe" fixes to longstanding issues.
```